jenkinsbuilder/project-defaults.yml

```diff
@@ -17,7 +17,7 @@
     builders:
         - shell: |
             export DOCKERFILE_DIR=`pwd`
-            TEST_TAG=`date | md5sum | base64 | head -c 14`
+            TEST_TAG=`date +%s%N | md5sum | base64 | head -c 14`
```

Comment:
@rtnpro That's a shell command. Not a python thing. Doint that would lead to:
```bash
$ date + %s%N
date: extra operand ‘%s%N’
Try 'date --help' for more information.
```